<ui version="4.0" >
 <class>webcamConfigWidget</class>
 <widget class="QWidget" name="webcamConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <property name="sizeConstraint" >
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="pictureLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>320</width>
         <height>240</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="text" >
        <string>PictureLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="_2" >
     <item>
      <layout class="QHBoxLayout" name="_3" >
       <property name="sizeConstraint" >
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QComboBox" name="comboBoxDevs" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy" >
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxSize" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy" >
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text" >
           <string>Minimal</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxFormat" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy" >
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="_5" >
       <item>
        <widget class="QGroupBox" name="controlsGroupBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Controls</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Sharpness</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sharpManualValueBox" />
            </item>
            <item>
             <widget class="QPushButton" name="sharpDefaultValueBtn" >
              <property name="text" >
               <string>Default</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Brightness</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="brightManualValueBox" >
              <property name="maximum" >
               <number>255</number>
              </property>
              <property name="value" >
               <number>127</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="brightDefaultValueBtn" >
              <property name="text" >
               <string>Default</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>Saturation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="satManualValueBox" >
              <property name="maximum" >
               <number>255</number>
              </property>
              <property name="value" >
               <number>39</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="satDefaultValueBtn" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Default</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" >
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Contrast</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="contManualValueBox" >
              <property name="maximum" >
               <number>255</number>
              </property>
              <property name="value" >
               <number>32</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="contDefaultValueBtn" >
              <property name="text" >
               <string>Default</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="freqBox" >
            <property name="text" >
             <string>Correct NÃ©on</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
